tests/test_github.py

```diff
@@ -0,0 +1,746 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2015-2016 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#     Quan Zhou <quan@bitergia.com>
+#
+
+import argparse
+import datetime
+import shutil
+import sys
+import tempfile
+import unittest
+
+import httpretty
+import requests
+
+if not '..' in sys.path:
+    sys.path.insert(0, '..')
+
+from perceval.cache import Cache
+from perceval.errors import BackendError, CacheError, ParseError
+from perceval.backends.github import GitHub, GitHubCommand, GitHubClient
+
+
+GITHUB_API_URL = "https://api.github.com"
+GITHUB_ISSUES_URL = GITHUB_API_URL + "/repos/zhquan_example/repo/issues"
+GITHUB_LOGIN_URL = GITHUB_API_URL + "/users/zhquan_example"
```

Comment:
Change the name of this constant to `GITHUB_USER_URL`. When I read it I think that it is an URL to sign in.